### DISCUSSION TITLE
Strange navigation on episode 220

### DISCUSSION THREAD

**Discussion by miibpa** [pointfreeco/episode-code-samples#132] [General]
Hi! I found a strange behavior on las episode's Standups demo, in some cases when saving a meeting from the end meeting button in the record meeting toolbar it navigates back to the Standups's list.  the steps to reproduce it is Create a new StandUp -> Start a meeting -> Forward to last speaker -> Save meeting in alert -> Create a new meeting -> End meeting from toolbar's end meeting button. 
I did not find out why is this happening.

Thanks in advance, and congrats for your fantastic work!!

I attach a video showing the issue:

https://user-images.githubusercontent.com/17158118/213394632-42d035c0-5153-43e7-8f9b-54279a00cc86.mov

**Comment by mbrandonw** [comment]
Hi @miibpa, unfortunately this is just due to more bugs in SwiftUI. 😕 There's a discussion about this going on over in our swiftui-navigation library:

https://github.com/pointfreeco/swiftui-navigation/issues/65#issuecomment-1383213290

I'm going to convert this to a discussion too.

**Reply by miibpa** [reply]
Thanks for your reply @mbrandonw! I'll keep an eye on this, hopefully Apple will fix this bugs soon 🙏
